engines/python/setup/djl_python/tests/test_rolling_batch.py

```diff
@@ -231,7 +231,12 @@ def test_sse_tgi_compat_fmt(self):
                 "text": " world",
                 "logprob": -0.567854
             },
-            "generated_text": "Hello world"
+            "generated_text": "Hello world",
+            "details": {
+                "finish_reason": "length",
+                "generated_tokens": 3,
+                "inputs": "This is a wonderful day"
+            }
```

Comment:
@lanking520 I believe, if tgi_compat is True, then we should always send out details even if it is not explicitly set. Previously we were not sending the details.